Makefile.sp

```diff
@@ -110,10 +110,13 @@ sp-vale: sp-install
 	@. $(VENV); test -d $(SPHINXDIR)/venv/lib/python*/site-packages/vale || pip install vale
 	@. $(VENV); test -f $(SPHINXDIR)/vale.ini || python3 $(SPHINXDIR)/get_vale_conf.py
 	@. $(VENV); find $(SPHINXDIR)/venv/lib/python*/site-packages/vale/vale_bin -size 195c -exec vale --config "$(SPHINXDIR)/vale.ini" $(TARGET) > /dev/null \;
+	@cat .sphinx/styles/config/vocabularies/Canonical/accept.txt > .sphinx/styles/config/vocabularies/Canonical/accept_backup.txt
+	@cat .wordlist.txt .custom_wordlist.txt >> .sphinx/styles/config/vocabularies/Canonical/accept.txt
```

Comment:
These file paths might not be in the root folder. 
By default, `$(SPHINXDIR)` is `.sphinx`, and `$(SOURCEDIR)` is `.`.

Reply:
```suggestion
	@cat $(SPHINXDIR)/styles/config/vocabularies/Canonical/accept.txt > $(SPHINXDIR)/styles/config/vocabularies/Canonical/accept_backup.txt
	@cat $(SOURCEDIR).wordlist.txt .custom_wordlist.txt >> $(SPHINXDIR)/styles/config/vocabularies/Canonical/accept.txt
```